pygn_server.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Python 3 is not always installed as `python3`, but I'm not requesting a change.  Things are just not consistent.

Reply:
I'll drop this. On ubuntu, `python` is explicitly python2, as far as I know.